tests/jobs/design_kw/test_design_kw.py

```diff
@@ -29,6 +29,17 @@ def test_extract_key_value_ok():
     }
 
 
+def test_extract_key_value_whitespace_error():
+    """Test that user friendly error message is displayed
+    when an error occurs"""
+    data = ['"FOO BAR COM " 2']
+    try:
```

Comment:
I prefer using: 
`with pytest.raises(ValueError) as error_msg:` in tests, though they do the same under the hood, it makes it easier to read (I think).

Reply:
Solved when I found the `match=` argument to `pytest.raises()`. If not, it is difficult to do asserts on the message.